starknet_devnet/starknet_wrapper.py

```diff
@@ -55,6 +59,12 @@ def __init__(self):
         self.__l1_provider = None
         """Saves the L1 URL being used for L1 <> L2 communication."""
 
+    @classmethod
+    def load(cls, path: str) -> "StarknetWrapper":
```

Comment:
Why is this a class method?

+ Is wrapping the return type in `"` a standard way of using the yet to be defined types?

Reply:
It doesn't use any properties of the class/object, so it doesn't make sense for it to be a regular method. It could be a standalone function, but I saw it as related to the `StarknetWrapper` class.

Reply:
And to answer the second question, that is the standard way.

Reply:
My question is more on why is it a class method and not static method?

Reply:
Made it a static method: https://github.com/Shard-Labs/starknet-devnet/pull/40/commits/a413e0a563826681652aebdd18c1c3449e75692a